src/pybind/mgr/cephadm/module.py

```diff
@@ -1839,18 +1839,25 @@ def describe_service(self, service_type=None, service_name=None,
                     continue
                 if dd.daemon_type == 'osd':
                     continue                # ignore OSDs for now
-                spec = None
                 if dd.service_name() in self.spec_store.specs:
                     spec = self.spec_store.specs[dd.service_name()]
+                else:
+                    spec = ServiceSpec(
+                        unmanaged=True,
+                        service_type=dd.daemon_type,
+                        service_id=n,
+                        placement=PlacementSpec(
+                            hosts=[dd.hostname]
+                        )
```

Comment:
this is actually questionable. after running

`ceph orch ls  --format yaml | ceph orch apply -i -´ 

this ends up in the store as `unmanged`